recipe/meta.yaml

```diff
@@ -28,8 +28,7 @@ requirements:
     - python
     - setuptools
     - cython
-    - numpy 1.13.3  # [py<37]
-    - numpy 1.14.5  # [py>=37]
+    - numpy
```

Comment:
Forgot to bump the build number. No big deal b/c I moved the previous `0` to broken and the hash is different, but it is prefer to always bump for new packages with the same version.

Reply:
I did do it in this commit https://github.com/conda-forge/scipy-feedstock/pull/100/commits/f560d4d85174b480d4495ca5995bcc33a967bbc8. Must have got lost somehow.

Reply:
Sorry about that. I was editing in GitHub and it got lost

Reply:
Not a problem. I do that all the time when editing online :smile: